.travis.yml

```diff
@@ -1,5 +1,15 @@
 language: node_js
 node_js:
   - "0.10"
-  - "0.8"
-  - "0.6"
+  - "4"
+  - "lts/*"
```

Comment:
we should probably add `node` to the versions to get the [latest stable version](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Specifying-Node.js-versions) (Node 9.x right now)

Reply:
Added. I wanna leave the lts in there too since it is required by the selenium-webdriver which is something I'm working with to do cross-browser tests on Browserstack :wink:

Reply:
Ya the minimum I always test is `lts/*` and `node`